boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts

```diff
@@ -226,3 +231,11 @@ zephyr_udc0: &usb1 {
 &gpt_hw_timer {
 	status = "okay";
 };
+
+&iomuxcgpr {
+	status = "okay";
+};
+
+&pinctrl {
+	status = "okay";
+};
```

Comment:
nit: since pinctrl is required by most (if not all) applications, you may enable this by default

Reply:
Fixed in latest push